src/formatter-settings/assets/style.scss

```diff
@@ -0,0 +1,221 @@
+  font-size: var(--vscode-font-size);
+  line-height: 1.4em;
+  padding-left: 27px !important;
+}
+
+.dropdown {
+  width: 250px !important;
+  .dropdown-button {
+    color: var(--vscode-settings-dropdownForeground) !important;
+    background-color: var(--vscode-settings-dropdownBackground) !important;
+    border: 1px solid var(--vscode-settings-dropdownBorder) !important;
+    padding: 2px 4px !important;
+    width: 100%;
+    &:focus, &:active {
+      outline: none !important;
+      box-shadow: none !important;
+      border: 1px solid var(--vscode-focusBorder) !important;
+    }
+  }
+  .dropdown-menu {
+    width: 100%;
+    border: 1px solid var(--vscode-focusBorder) !important;
+    background-color: var(--vscode-settings-dropdownBackground) !important;
+    background-clip: unset !important;
+  }
+  .dropdown-item {
+    background-color: var(--vscode-settings-dropdownBackground) !important;
+    color: var(--vscode-settings-dropdownForeground) !important;
+  
+    &:hover {
+      background-color: var(--vscode-list-activeSelectionBackground) !important;
+      color: var(--vscode-list-activeSelectionForeground) !important;
+    }
+  }
+  .dropdown-toggle {
+    &:after {
+      display: none !important;
+    }
+  }
+  svg.codicon {
+    color: var(--vscode-settings-textInputForeground);
+  }
+}
+
+.flex-vertical-center {
+  display: flex !important;
+  justify-content: space-between !important;
+  align-items: center !important;
+}
+
+.form-label {
+  margin-top: 0 !important;
+  margin-bottom: 0 !important;
+}
+
+.form-check {
+  position: relative !important;
+  min-height: 1.5rem !important;
+  padding-left: 0px !important;
+  .form-check-input {
+    opacity: 0;
+    &:checked~.form-check-label svg.codicon {
+      display: block;
+    }
+    &:focus~.form-check-label::before {
+      outline: none;
+      box-shadow: none;
+      border: 1px solid var(--vscode-focusBorder);
+    }
+  }
+
+  .form-check-label {
+    position: relative;
+    vertical-align: top;
+    line-height: 1.4em !important;
+    font-size: var(--vscode-font-size) !important;
+    svg.codicon {
+      position: absolute;
+      color: var(--vscode-settings-textInputForeground);
+      width: 18px;
+      height: 18px;
+      display: none;
+    }
+    &::before{
+      float: left !important;
+      display: block;
+      width: 18px;
+      height: 18px;
+      margin-right: 9px;
+      color: var(--vscode-settings-textInputForeground);
+      background-color: var(--vscode-settings-textInputBackground);
+      border: 1px solid var(--vscode-settings-textInputBorder);
+      border-radius: 3px 3px 3px 3px;
+      content: "";
+    }
+  }
+}
+
+.form-control {
+  color: var(--vscode-settings-textInputForeground) !important;
+  background-color: var(--vscode-settings-textInputBackground) !important;
+  border: 1px solid var(--vscode-settings-textInputBorder) !important;
+  height: auto !important;
+  padding: 2px 4px !important;
```

Comment:
Can we use the padding utility class provided by Bootstrap?

Reply:
If we want to use Bootstrap utility here, we have to define a some `$spacer` mentioned in [Spacing-Bootstrap](https://getbootstrap.com/docs/4.4/utilities/spacing/), since the css has no basic padding or margin, the values are like "2px", "3px", "12px" and so on, we just define some Enum values like this?
```
$spacers: (
  0: 0,
  1: 2px,
  2: 3px,
  3: 4px,
  4: 9px,
  5: 10px,
  6: 12px
)
```